02-overview-surveys.Rmd

```diff
@@ -149,39 +150,39 @@ To avoid this issue, researchers should consider these possibilities and adjust
+Before data is made publicly available, researchers will need to ensure that individual respondents can not be identified by the data when confidentiality is required. There are a variety of different methods that can be used, including data swapping, top or bottom coding, coarsening, and perturbation. In data swapping, researchers may swap specific data values across different respondents so that it does not impact insights from the data but ensures that specific individuals cannot be identified. For extreme values, top and bottom coding is sometimes used. For example, researchers may top-code income values such that households with income greater than \$99,999,999 are coded into a single category of \$99,999,999 or more. Other disclosure methods may include aggregating response categories or location information to avoid having only a few respondents in a given group and thus be identified. For example, researchers may use coarsening to display income in categories instead of as a continuous variable. Data producers may also perturb the data by adding random noise. There is as much art as there is a science to the methods used for disclosure, and in documentation, researchers should only provide high-level comments that disclosure was conducted and not specific details to ensure nobody can reverse the disclosure and thus identify individuals. For more information on different disclosure methods, please see {CITE Skinner chapter in this book: https://www.sciencedirect.com/science/article/abs/pii/S0169716108000151
 AAPOR checklist/standards: https://www-archive.aapor.org/Standards-Ethics/AAPOR-Code-of-Ethics/Survey-Disclosure-Checklist.aspx}.
 
 ## This book
 
-After all of these steps are taken, the data is ready for use by analysts. The rest of this book works from this point. If you're interested in learning more about the steps talked about here, we recommend you look into the references cited throughout this chapter.
```

Comment:
I think we can add a bit more about this paragraph since this is part of the survey life cycle:
* Documentation
* Data Analysis and Reporting
It can mirror the chapters of the book, as well.